e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -0,0 +1,512 @@
+package gov.cms.ab2d.e2etest;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+
+// Unit tests here can be run from the IDE and will use LOCAL as the default, they can also be run from the TestLauncher
+// class to specify a custom environment
+@Testcontainers
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ExtendWith(TestRunnerParameterResolver.class)
+public class TestRunner {
+
+    private static HttpClient httpClient;
+
+    private static String AB2D_API_URL;
+
+    private static String AB2D_ADMIN_URL;
+
+    private static final String PATIENT_EXPORT_PATH = "Patient/$export";
+
+    private static String jwtStr = null;
+
+    private static String TEST_USER;
+
+    private static final long DEFAULT_TIMEOUT = 30;
+
+    private static final int DELAY = 30;
+
+    private static final int JOB_TIMEOUT = 300;
+
+    private Environment environment;
+
+    public void runTests() throws InterruptedException, SQLException, JSONException, IOException {
+        runSystemWideExport();
+        runContractNumberExport();
+        testDelete();
+        //stressTest();
+    }
+
+    public TestRunner(Environment environment) throws IOException, InterruptedException, JSONException {
+        this.environment = environment;
+        init();
+    }
+
+    public void init() throws IOException, InterruptedException, JSONException {
+        if(environment.isUsesDockerCompose()) {
+            DockerComposeContainer container = new DockerComposeContainer(
+                    new File("../docker-e2e-compose.yml"))
+                    //.withScaledService("api", 2) // failing now since it's not changing ports
+                    //.withScaledService("worker", 2)
```

Comment:
Can we still scale the worker though, since it is not using a port? If yes, let's definitely do so.

Reply:
Yes the worker can be changed to 2